Sources/ReviewKit/ReviewKit.swift

```diff
@@ -15,7 +15,7 @@ public enum LayoutType {
     case full, score, graphical
 }
 
-public struct ShapeProgressView: View {
```

Comment:
We should keep this view but deprecate it with the @available macro. Add the new name as duplicate views this way we give users time to update. You should be fine to rename the child views though since those are not accessed by the user.

Reply:
I will do so. ✌🏾